test/fixtures/bind-carousel.html

```diff
@@ -16,9 +16,8 @@
   <p id="slideNumber" [text]="selectedSlide">0</p>
   <amp-carousel width="300" height="100" type="slides" id="carousel"
       on="slideChange:AMP.setState({selectedSlide: event.index})" [slide]="selectedSlide">
-    <amp-img src="http://www.google.com/image1" height="200" width="200"></amp-img>
-    <amp-img src="http://www.google.com/image1" height="200" width="200"></amp-img>
-    <amp-img src="http://www.google.com/image1" height="200" width="200"></amp-img>
+    <amp-img src="http://www.foo.com/image" height="200" width="200"></amp-img>
+    <amp-img src="http://www.foo.com/image" height="200" width="200"></amp-img>
```

Comment:
Real, locally served images would avoid polluting the error log but it doesn't affect test correctness either way. Would be a good clean-up task or GFI.